src/mpyl/steps/deploy/kubernetes.py

```diff
@@ -66,7 +66,12 @@ def execute(self, step_input: Input) -> Output:
                     f"Service {step_input.project.name} reachable at: {hostname}"
                 )
                 endpoint = (
-                    "/" if has_specific_routes_configured else "/swagger/index.html"
+                    "/"
+                    if (
+                        has_specific_routes_configured
+                        or "nginx" in step_input.project.name
```

Comment:
Did you purposely use a 'contains' check or did you want to use an 'equals' (`==`)? Looks good otherwise!